cmd/process-agent/collector.go

```diff
@@ -336,3 +339,32 @@ func isHTTPTimeout(err error) bool {
 	}
 	return false
 }
+
+func hasContainers(body []byte) bool {
```

Comment:
we should find a way to call this on a MessageBody instead of a `[]byte`. That way we don't have to do an unnecessary `DecodeMessage`  to figure out if there are containers.

Reply:
+1 here :)